contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -2813,4 +2871,662 @@ boolean GetVolumePathNamesForVolumeName(String lpszVolumeName,
+	 *            Alternately, rather than a pointer, this parameter can be
+	 *            MAKEINTRESOURCE(ID), where ID is the integer identifier of the
+	 *            given resource type.<br>
+	 *            For standard resource types, see Resource Types. For more
+	 *            information, see the Remarks section below.
+	 * @return If the function succeeds, the return value is a handle to the
+	 *         specified resource's information block.<br>
+	 *         To obtain a handle to the resource, pass this handle to the
+	 *         LoadResource function. <br>
+	 *         If the function fails, the return value is NULL.<br>
+	 *         To get extended error information, call GetLastError.
+	 */
+	HRSRC FindResource(HMODULE hModule, Pointer name, Pointer type);
+
+	/**
+	 * Retrieves a handle that can be used to obtain a pointer to the first byte
+	 * of the specified resource in memory.
+	 * 
+	 * @param hModule
+	 *            A handle to the module whose executable file contains the
+	 *            resource. <br>
+	 *            If hModule is NULL, the system loads the resource from the
+	 *            module that was used to create the current process.
+	 * @param hResource
+	 *            A handle to the resource to be loaded. <br>
+	 *            This handle is returned by the FindResource or FindResourceEx
+	 *            function.
+	 * @return If the function succeeds, the return value is a handle to the
+	 *         data associated with the resource.<br>
+	 *         If the function fails, the return value is NULL. <br>
+	 *         To get extended error information, call GetLastError.
+	 */
+	HANDLE LoadResource(HMODULE hModule, HRSRC hResource);
+
+	/**
+	 * Retrieves a pointer to the specified resource in memory.
+	 * 
+	 * @param hResource
+	 *            A handle to the resource to be accessed. <br>
+	 *            The LoadResource function returns this handle.<br>
+	 *            Note that this parameter is listed as an HGLOBAL variable only
+	 *            for backward compatibility.<br>
+	 *            Do not pass any value as a parameter other than a successful
+	 *            return value from the LoadResource function.
+	 * @return If the loaded resource is available, the return value is a
+	 *         pointer to the first byte of the resource; otherwise, it is NULL.
+	 */
+	Pointer LockResource(HANDLE hResource);
+
+	/**
+	 * @param hModule
+	 *            A handle to the module whose executable file contains the
+	 *            resource.
+	 * @param hResource
+	 *            A handle to the resource. This handle must be created by using
+	 *            the FindResource or FindResourceEx function.
+	 * @return If the function succeeds, the return value is the number of bytes
+	 *         in the resource. <br>
+	 *         If the function fails, the return value is zero. To get extended
+	 *         error information, call GetLastError.
+	 */
+	DWORD SizeofResource(HMODULE hModule, HANDLE hResource);
```

Comment:
_DWORD_ as a return type is simply an _int_ - if you fear there are some sign issues then return a _long_ - I believe it should work.